tests/TESTINS5.MLC

```diff
@@ -341,6 +375,215 @@ BIC_OK2  EQU    *
          IILL   R3,4
          CLG    R3,=AL2(1,2,3,4)
          RT1    CC0
+* MGRK #300
+         LMG   R5,R6,=FL8'2,3'
+         MGRK  R2,R5,R6
+         STMG  R2,R3,DWORK
+         CLC   DWORK,=FL16'6'
+         RT1    CC0 
+         LMG   R5,R6,=FL8'2,-3'
+         MGRK  R2,R5,R6
+         STMG  R2,R3,DWORK
+         CLC   DWORK,=FL16'-6'
+         RT1    CC0 
+         LMG   R5,R6,=FL8'-2,3'
+         MGRK  R2,R5,R6
+         STMG  R2,R3,DWORK
+         CLC   DWORK,=FL16'-6'
+         RT1    CC0 
+         LMG   R5,R6,=FL8'-2,-3'
+         MGRK  R2,R5,R6
+         STMG  R2,R3,DWORK
+         CLC   DWORK,=FL16'6'
+         RT1    CC0 
+         LMG   R5,R6,=FL8'123456,10000000'
+         MGRK  R2,R5,R6
+         STMG  R2,R3,DWORK
+         CLC   DWORK,=FL16'1234560000000'
+         RT1    CC0 
+* MS
+         L      R1,=F'2'
+         MS    R1,=F'3'
+         CL     R1,=F'6'
+         RT1  CC0
+         L       R1,=F'2'
+         MS    R1,=F'-3'
+         CL     R1,=F'-6'
+         RT1  CC0
+         L       R1,=F'-2'
+         MS    R1,=F'3'    #300 ADDED TEST
+         CL     R1,=F'-6'
+         RT1  CC0
+         L       R1,=F'-2'
+         MS    R1,=F'-3'
+         CL     R1,=F'6'
+         RT1  CC0
+* MSY
+         L      R1,=F'2'
+         MSY    R1,=F'3'
+         CL     R1,=F'6'
+         RT1  CC0
+         L       R1,=F'2'
+         MSY    R1,=F'-3'
+         CL     R1,=F'-6'
+         RT1  CC0
+         L       R1,=F'-2'
+         MSY    R1,=F'3'  #300 ADDED TEST
+         CL     R1,=F'-6'
+         RT1  CC0
+         L       R1,=F'-2'
+         MSY    R1,=F'-3'
+          CL     R1,=F'6'
+          RT1  CC0
+* MSGRKC
+         LMG   R5,R6,=FL8'2,3'
+         SR       R0,R0
+         MSGRKC  R2,R5,R6
+         RT1 CC2
+         CLG    R2,=FL8'6'
+         RT1    CC0 
+         LMG   R5,R6,=FL8'2,-3'
+         SR       R0,R0
+         MSGRKC  R2,R5,R6
+         RT1    CC1
+         CLG    R2,=FL8'-6'
+         RT1    CC0 
+         LMG   R5,R6,=FL8'-2,3'
+         SR       R0,R0
+         MSGRKC  R2,R5,R6
+         RT1     CC1
+         CLG    R2,=FL8'-6'
+         RT1    CC0 
+        LMG   R5,R6,=FL8'0,3'
+         SR      R0,R0
+         AHI    R0,1    NOT ZERO     
+         MSGRKC  R2,R5,R6
+         RT1     CC0
+         CLG    R2,=FL8'0'
+         RT1    CC0 
+         LMG   R5,R6,=FL8'3,0'
+         SR      R0,R0
+         AHI    R0,1    NOT ZERO     
+         MSGRKC  R2,R5,R6
+         RT1     CC0
+         CLG    R2,=FL8'0'
+         RT1    CC0 
+         LMG   R5,R6,=FL8'0,0'
+         SR      R0,R0
+         AHI    R0,1    NOT ZERO     
+         MSGRKC  R2,R5,R6
+         RT1     CC0
+         CLG    R2,=FL8'0'
+         RT1    CC0 
+         LMG   R5,R6,=FL8'-2,-3'
+         SR       R0,R0
+         MSGRKC  R2,R5,R6
+         RT1   CC2
+         CLG   R2,=FL8'6'
+         RT1    CC0 
```

Comment:
Nothing wrong here. Just suggesting that you move this test up as the last of the other "+/-2,+/-3" tests.